### ISSUE TITLE
Number of rows argument in tribble_paste

### ISSUE THREAD

**Issue by pgensler** [MilesMcBain/datapasta#65]
It could be useful to include an argument for number of rows to output, so if you wanted to only have 5 or 10 rows from a larger dataset.

**Comment by pgensler** [comment]
Thinking about this, maybe having a `tribble_paste` where it defaults to the first 5 rows of your dataframe would help to improve reprex's?

**Comment by pgensler** [comment]
Yeah, maybe having a twitter poll could help to figure out what that number should be, or asking in the R4DS online community could be a good starting point?

**Comment by jfunction** [comment]
Isn't this what `head` is for?
